recipes/fortnet/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "fortnet" %}
+{% set version = "0.7.1" %}
+{% set build = 0 %}
+{% set mpi = mpi or "nompi" %}
+{% set origin = "vanderhe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ origin }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a7537585e947e3352324b866533b7756a6cfb5ff1524e3a2ff16929d9283cb39
+
+{% if mpi == "nompi" %}
+{% set build = build + 100 %}
+{% endif %}
+build:
+  number: {{ build }}
+  skip: True  # [not linux]
+
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+
+  {% if mpi != "nompi" %}
+  run_exports:
+    - {{ name }} * {{ mpi_prefix }}_*
+  {% endif %}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - python
+    - h5py
+    - numpy
+    - cmake
+    - ninja
+    - fypp
+    - wget  # [build_platform == target_platform]
+    - hdf5 >=1.10.4  # [build_platform != target_platform]
+    - hdf5 * nompi_*  # [build_platform != target_platform]
+  host:
+    - {{ mpi }}  # [mpi != "nompi"]
+    - mpifx * mpi_{{ mpi }}_*  # [mpi != "nompi"]
+    - libblas
+    - liblapack
```

Comment:
This package has multiple [warnings about overdepending](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=598957&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2239). Please remove liblapack if it is not linked (static linking is not allowed).

Also, it's strange that there is a non-link warning about mpich and openmpi too.

Reply:
I've also seen the warnings, but do not understand them, as linking without e.g. liblapack dependency fails (tried again, just to be sure). Same confusion about mpich/openmpi.